mmv1/third_party/terraform/services/backupdr/data_source_backup_dr_management_server.go.erb

```diff
@@ -0,0 +1,104 @@
+<% autogen_exception -%>
+package backupdr
+<% unless version == 'ga' -%>
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+	"strings"
+)
+
+func DataSourceGoogleCloudBackupDRService() *schema.Resource {
+
+	dsSchema := tpgresource.DatasourceSchemaFromResourceSchema(ResourceBackupDRManagementServer().Schema)
+	tpgresource.AddRequiredFieldsToSchema(dsSchema, "location")
+
+	return &schema.Resource{
+		Read:   dataSourceGoogleCloudBackupDRServiceRead,
+		Schema: dsSchema,
+	}
+}
+func flattenBackupDRManagementServerName(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
+	return v
+}
+
+func flattenBackupDRManagementServerResourceResp(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) map[string]interface{} {
+	if v == nil {
+		fmt.Printf("Interface is nil: %s", v)
+	}
+	fmt.Printf("Interface is : %s", v)
+	l := v.([]interface{})
+	for _, raw := range l {
+		return flattenBackupDRManagementServerResource(raw, d, config)
```

Comment:
sorry, I'm not sure if I understand it 100%, are we only needing the first response?

Reply:
Yes, we have only one resource returned as part of response so far, as our resource is unique to consumer project. Array is sent since it is list call from CCFE, also in future we may support region wise creation.

Reply:
Confirmed with @niharika-98 offline, there's only one resource per location per project, so the list will return the one and only resource which is the one needed. 

Would you mind leaving an inline comment states that we're only returning the first element and why that's intended. Thanks!